content/docs/static-type-checking.md

```diff
@@ -51,53 +51,53 @@ Now, add `flow` to the `"scripts"` section of your `package.json` to be able to
 }
 ```
 
-Finally, run one of the following commands:
+Son olarak, aşağıdaki komutlardan birini çalıştırın:
 
-If you use [Yarn](https://yarnpkg.com/), run:
+Eğer [Yarn](https://yarnpkg.com/) kullanıyorsanız, bu komutu çalıştırın:
 
 ```bash
 yarn run flow init
 ```
 
-If you use [npm](https://www.npmjs.com/), run:
+Eğer [npm](https://www.npmjs.com/) kullanıyorsanız, bu komutu çalıştırın:
 
 ```bash
 npm run flow init
 ```
 
-This command will create a Flow configuration file that you will need to commit.
+Bu komut, sizin için commit etmeniz gereken bir Flow ayar dosyası oluşturacaktır.
 
-### Stripping Flow Syntax from the Compiled Code {#stripping-flow-syntax-from-the-compiled-code}
+### Flow Sözdiziminin Derlenmiş Koddan Ayrılması {#stripping-flow-syntax-from-the-compiled-code}
 
-Flow extends the JavaScript language with a special syntax for type annotations. However, browsers aren't aware of this syntax, so we need to make sure it doesn't end up in the compiled JavaScript bundle that is sent to the browser.
+Flow, JavaScript dilini tip açıklamaları için özel bir sözdizimi yardımıyla genişletir. Ancak, tarayıcılar bu sözdiziminden haberdar değildir. Bu yüzden, onun tarayıcıya yollanan JavaScript demetinde yer almadığından emin olmalıyız.
 
-The exact way to do this depends on the tools you use to compile JavaScript.
+Bunun tam olarak nasıl sağlandığı, JavaScript'i derlemek için kullandığınız araca bağlıdır.
 
 #### Create React App {#create-react-app}
 
-If your project was set up using [Create React App](https://github.com/facebookincubator/create-react-app), congratulations! The Flow annotations are already being stripped by default so you don't need to do anything else in this step.
+Eğer projeniz [Create React App](https://github.com/facebookincubator/create-react-app) ile oluşturulduysa, tebrikler! Flow açıklamaları zaten öntanımlı olarak saf dışı bırakılmıştır. Bu yüzden sizin bu adımda herhangi bir şey yapmanıza gerek kalmaz.
```

Comment:
`saf disi birakmak` sounds like a negative meaning here. Like disabling Flow. But it is not actually. What do you think if we use `ayrılmak` in this case? Since you already used the word `Ayrılması` in previous title, this might more sense. Check this, please:

> Flow açıklamaları zaten öntanımlı olarak ayrılmıştır.